src/pl.x86.json

```diff
@@ -69,7 +69,7 @@
 			"version": "2.0.0.0",
 			"id": "30d7324fd390fa336dc9ec5a249d69c825b5e59c1784097a6fdc291f9a332187",
 			"repository": "https://github.com/oleg-shilo/scripts.npp/releases/download/v2.0.0.0/NppScripts.x86.zip",
-			"description": "Notepad++ plugin for C# based automation. The plugin allows implementing a simple macros-like automations as well as the full scale script based plugins by means of C# scripts. The solution is based on CS-Script C# script engine.",
+			"description": "C# based automation.  Allows implementing a simple macros-like automations as well as the full scale script based plugins by means of C# scripts. The solution is based on CS-Script C# script engine.",
```

Comment:
A superfluous space in front of `Allows`.

Reply:
Space removed.